hazelcast/src/main/java/com/hazelcast/cluster/impl/TcpIpJoiner.java

```diff
@@ -71,6 +90,9 @@ public TcpIpJoiner(Node node) {
         }
         maxPortTryCount = tryCount;
         joinConfig = getActiveMemberNetworkConfig(config).getJoin();
+        previouslyJoinedMemberAddressRetentionDuration = TimeUnit.SECONDS.toMillis(
```

Comment:
then here you have just Duration.ofSeconds

Reply:
same as above